experiments/causal_pipeline/bash_scripts/run_experiment.sh

```
@@ -0,0 +1,179 @@
+            echo "  - $filename"
+        fi
+    done
+    echo ""
+    echo "Example: ./run_experiment.sh ce0_cy0_y0_i0"
+    read -p "Press Enter to continue..."
+    exit 1
+fi
+
+EXPERIMENT_NAME=$1
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+CONFIG_DIR="$SCRIPT_DIR/generated_configs/$EXPERIMENT_NAME"
```

Shellcheck (0.11.0)

[warning] CONFIG_DIR appears unused. Verify use (or export if used externally).

(SC2034)